### ISSUE TITLE
build failed used Ubuntu

### ISSUE THREAD

**Issue by cainoftes** [esmini/esmini#416]
I am building esmini by Ubuntu shell and i reference the [https://esmini.github.io/index.html#_build_esmini_quick_guide](https://esmini.github.io/index.html#_build_esmini_quick_guide) guild.
But when i executed the command of `cmake ..`, a CMake Error occur. The relative infomation show in here
```
CMake Error at support/cmake/cloud/download.cmake:21 (message):
  FAILED to download osg.7z (Status: 6;"Couldn't resolve host name") - Could
  not resolve host: esmini.asuscomm.com

  Closing connection 0

Call Stack (most recent call first):
  support/cmake/cloud/download.cmake:61 (download_and_extract)
  CMakeLists.txt:171 (download)
```

**Comment by eknabevcc** [comment]
Yes, there seems to be a temporary problem with the esmini storage.

Meanwhile, you can try to change server settings in [CMakeLists.txt](https://github.com/esmini/esmini/blob/master/CMakeLists.txt) (under esmini root), for example referring to dropbox instead of esmini:
```
set(FILE_STORAGE
    "dropbox") # "dropbox", "google", "esmini"  (limited GB/Day)
set(MODEL_STORAGE
    "dropbox") # "dropbox", "google", "esmini"  (limited GB/Day)
```

Then to apply settings:
```
cd build
rm CMakeCache.txt
cmake ..
```

**Comment by cainoftes** [comment]
Thank you very much! when i modify the CMakeLists.txt and alter esmini to dropbox or google. The command worked.
By the way, the modified location is line 138 and line 140 in CMakeLists.txt.
